modules/lvgl/input/lvgl_common_input.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 Fabian Blatz <fabianblatz@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/device.h>
+#include <zephyr/input/input.h>
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/display.h>
+
+#include "lvgl_common_input.h"
+#include <zephyr/logging/log.h>
+LOG_MODULE_DECLARE(lvgl);
```

Comment:
```suggestion
#include "lvgl_common_input.h"

#include <zephyr/device.h>
#include <zephyr/kernel.h>
#include <zephyr/logging/log.h>

LOG_MODULE_DECLARE(lvgl);
```

please make sure to sort headers properly (local, zephyr, 3rd party) and do not include stuff that is not needed.